frontend_tests/node_tests/composebox_typeahead.js

```diff
@@ -65,9 +66,61 @@ const ct = composebox_typeahead;
 ct.__Rewire__("max_num_items", 15);
 
 const mention_all = ct.broadcast_mentions()[0];
+const mention_everyone = ct.broadcast_mentions()[1];
+const mention_stream = ct.broadcast_mentions()[2];
 assert.equal(mention_all.email, "all");
 assert.equal(mention_all.full_name, "all");
+assert.equal(mention_everyone.email, "everyone");
+assert.equal(mention_everyone.full_name, "everyone");
+assert.equal(mention_stream.email, "stream");
+assert.equal(mention_stream.full_name, "stream");
+
+run_test("verify wildcard mentions typeahead for stream message", () => {
+    const wildcard_string = $t({defaultMessage: "Notify stream"});
+
+    assert.equal(
+        mention_all.special_item_text,
+        $t(
+            {defaultMessage: "{wildcard_mention_token} ({wildcard_string})"},
+            {wildcard_mention_token: mention_all.email, wildcard_string},
+        ),
+    );
+    assert.equal(
+        mention_everyone.special_item_text,
+        $t(
+            {defaultMessage: "{wildcard_mention_token} ({wildcard_string})"},
+            {wildcard_mention_token: mention_everyone.email, wildcard_string},
+        ),
```

Comment:
This structure of test, where we compute variables, isn't very helpful.

What we want is to verify the output, e.g. `assert.equal(mention_all.special_item_text, "all (translated: Notify stream)");`